java/src/jmri/jmrix/can/cbus/swing/simulator/SimulatorPane.java

```diff
@@ -39,15 +40,10 @@ public class SimulatorPane extends jmri.jmrix.can.swing.CanPanel {
     @Override
     public void initComponents(CanSystemConnectionMemo memo) {
```

Comment:
THere's a null check and exception on `memo` later on.  Maybe an @NonNull here?

Reply:
Thanks and makes sense, I thought that the super initComponents had conflicting annotation but not the case :-)